include/micm/util/vector_matrix.hpp

```diff
@@ -228,6 +228,17 @@ namespace micm
       return L;
     }
 
+    void print() const {
+      for (std::size_t i = 0; i < x_dim_; ++i)
+      {
+        for (std::size_t j = 0; j < y_dim_; ++j)
+        {
+          std::cout << (*this)[i][j] << " ";
+        }
+        std::cout << std::endl;
+      }
+    }
+
```

Comment:
Doesn't look like it is used anywhere, could you confirm it? Since we don't execute clang tidy automatically, could you name the function in CamelCase?

Reply:
It was useful when I was debugging. I'd like to keep it

Reply:
Yeah that's fine with me. Would you be able to make it in CamelCase though for the consistency in our code base?